src/main/java/lotto/domain/LottoTicket.java

```diff
@@ -0,0 +1,86 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import lotto.exceptions.InvalidLottoTicketException;
+
+public class LottoTicket {
+    private static final int LOTTO_NUMBER_COUNT = 6;
+    private static final String DELIMITER = ", ";
+    private static final String PREFIX = "[";
+    private static final String SUFFIX = "]";
+    private final List<LottoNumber> lottoTicket;
+
+    private LottoTicket(final List<LottoNumber> lottoTicket) {
+        validate(lottoTicket);
+        this.lottoTicket = Collections.unmodifiableList(lottoTicket);
+    }
+
+    public LottoTicket(String[] numbers) {
+        List<LottoNumber> lottoNumbers = new ArrayList<>();
+        for (String number : numbers) {
+            lottoNumbers.add(LottoNumber.find(number.trim()));
+        }
+        validate(lottoNumbers);
+        this.lottoTicket = lottoNumbers;
```

Comment:
```suggestion
        this(lottoNumbers);
```

생성자에서 인스턴스 변수는 한 곳에서 해주는 것이 좋은데요.
어떠한 이유 때문인지 고민해보아요!